src/Entities/Schema.php

```diff
@@ -30,6 +30,15 @@
     public function __construct()
     {
         $this->builders = [
+            'bank-accounts' => function (array $content) {
+                return new Collection(new BankAccount(), $content);
+            },
+            'bank-accounts/{bankAccountId}' => function (array $content) {
+                return new BankAccount($content);
+            },
+            'bank-accounts/{bankAccountId}/deactivation' => function (array $content) {
```

Comment:
I think on this endpoint we cannot get any resource, so this builder is unnecessary.